app/src/main/java/eu/theinvaded/mastondroid/viewmodel/LoginViewModelContract.java

```diff
@@ -20,7 +20,7 @@
         String getClientSecret();
         String getClientId();
         void setUser(MastodonAccount account);
-        void showLoginError();
+        void showLoginError(Throwable loginError);
```

Comment:
Thanks, i'm kinda used to doing that in web apps, but not too familiar with Java yet, so i'd rather learn from how you do that. :)